nameko_grpc/connection.py

```diff
@@ -331,7 +380,14 @@ def send_request(self, request_headers):
         over the response.
 
         Invocations are queued and sent on the next iteration of the event loop.
+
+        raises ConnectionTerminatingError if connection is terminating. Check
```

Comment:
Haven't patched h2 well enough for it to know it's terminating and not accepting new streams so logic is required here. We don't hit this as we check if a connection is_alive before calling and create a new conn if the existing is terminating.

Reply:
Maybe elevate this note into a comment in the code, for posterity in case h2 fixes this problem properly.